app/src/main/java/ca/klapstein/baudit/Data/Password.java

```diff
@@ -2,19 +2,24 @@
 
 import android.support.annotation.NonNull;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import ca.klapstein.baudit.Managers.BauditRemoteManager;
+
 /**
  * Data class representing a Baudit's {@code User}'s password.
  *
  * @see User
  */
 public class Password {
     private static final String TAG = "Password";
+    private static final String VALID_PASSWORD = "[a-zA-Z0-9]{8,20}";
 
     @NonNull
     private String password;
 
     public Password(@NonNull String password) {
-        // TODO: validate password
```

Comment:
there should be a check before constructing a password.

What if someone creates a password with an incorrect password string?

This will constructor will still allow it to be set.

Reply:
added validation